pages/browse/index.js

```diff
@@ -46,6 +48,13 @@ const Browse = ({ session }) => {
     )
   }
 
+  const truncateDescription = (desc = '', maxLength, isOpen) => {
+    if (desc.length <= maxLength || isOpen) return { truncated: desc, cutOffIndex: desc.length }
+    const lastSpaceIndex = desc.substring(0, maxLength).lastIndexOf(' ')
+    const ellipsis = isOpen ? '' : '...'
+    return { truncated: desc.slice(0, lastSpaceIndex) + ellipsis, cutOffIndex: lastSpaceIndex }
+  }
+
```

Comment:
can this get moved to "utils/helpers.js"?

Reply:
yes definitely, i just tried putting it back there and it worked! for some reason when i first tried that it didn't want to work for me.. but it was probably some other confounding error